Makefile

```diff
@@ -48,18 +47,42 @@ endif
 export SELF_SIGNED_CERTS   ?= true
 export INSTALLATION_TYPE   ?= managed
 export INSTALLATION_NAME   ?= rhmi
-export INSTALLATION_PREFIX ?= redhat-rhmi
+
 export USE_CLUSTER_STORAGE ?= true
 export OPERATORS_IN_PRODUCT_NAMESPACE ?= false # e2e tests and createInstallationCR() need to be updated when default is changed
 export DELOREAN_PULL_SECRET_NAME ?= integreatly-delorean-pull-secret
 export ALERTING_EMAIL_ADDRESS = noreply-test@rhmi-redhat.com
 
+ifeq ($(INSTALLATION_TYPE), managed)
+	PROJECT=integreatly-operator
+	TAG ?= RHMI_TAG
+	OPERATOR_IMAGE=$(REG)/$(ORG)/$(PROJECT):v$(TAG)
+	NAMESPACE_PREFIX ?= redhat-rhmi-
+	APPLICATION_REPO ?= integreatly
+	export INSTALLATION_PREFIX ?= redhat-rhmi
+	export OLM_TYPE ?= integreatly-operator
+endif
+
+ifeq ($(INSTALLATION_TYPE), managed-api)
+	PROJECT=managed-api-service
+	TAG ?= RHOAM_TAG
+	OPERATOR_IMAGE=$(REG)/$(ORG)/$(PROJECT):v$(TAG)
+	NAMESPACE_PREFIX ?= redhat-rhoam-
+	APPLICATION_REPO ?= managed-api-service
+	export INSTALLATION_PREFIX ?= redhat-rhoam
```

Comment:
This is used in the path of file to set up the service account running locally. https://github.com/integr8ly/integreatly-operator/blob/master/Makefile#L73
In this case I believe we need it to be `redhat-managed-api` unless there are plans to rename the folder.

Reply:
Yes this was something I was miss understanding and it needs to be changed. I will also check the file for the any other cases where this is been used.